GITOPS-2269 - updated 4.9 release notes (#51684)
openshift/openshift-docs: modules/gitops-release-notes-1-6-1.adoc

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

:_content-type: REFERENCE

[id="gitops-release-notes-1-6-1_{context}"]
= Release notes for {gitops-title} 1.6.1

{gitops-title} 1.6.1 is now available on {product-title} 4.8 and above.

[id="fixed-issues-1-6-1_{context}"]
== Fixed issues

The following issues have been resolved in the current release:

> * Before this update, in a large set of applications the application controllers were restarted multiple times due to the unresponsiveness of liveness probes. This update fixes the issue by removing the liveness probe in the application controller `StatefulSet` object. link:https://issues.redhat.com/browse/GITOPS-2153[GITOPS-2153]

Reviewer: ```suggestion
* Before this update, the application controllers in a large set of applications were restarted multiple times due to the unresponsiveness of liveness probes. This update fixes the issue by removing the liveness probe in the application controller `StatefulSet` object. link:https://issues.redhat.com/browse/GITOPS-2153[GITOPS-2153]
```

> * Before this update, the RHSSO certificate cannot be validated when it is set up with a certificate which is not signed by certificate authorities. This update fixes the issue and now you can provide a custom certificate which will be used in verifying the Keycloak's TLS certificate when communicating with it. You can add the `rootCA` to the Argo CD custom resource `.spec.keycloak.rootCA` field. The Operator reconciles this change and updates the `oidc.config` field in the `argocd-cm` `ConfigMap` with the PEM-encoded root certificate. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]

Reviewer: ```suggestion
* Before this update, the RHSSO certificate could not be validated when it is set up with a certificate that was not signed by certificate authorities. This update fixes the issue and now you can provide a custom certificate that will be used in verifying the Keycloak's TLS certificate when communicating with it. You can add the `rootCA` to the Argo CD custom resource `.spec.keycloak.rootCA` field. The Operator reconciles this change and updates the `oidc.config` field in the `argocd-cm` `ConfigMap` with the PEM-encoded root certificate. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]
```
